backend/package.json

```diff
@@ -17,6 +17,18 @@
     "express-oauth2-jwt-bearer": "^1.3.0",
     "node": "^19.6.0",
     "nodemon": "^2.0.20",
-    "pg": "^8.8.0"
+    "pg": "^8.8.0",
+    
+    "@devexpress/dx-react-core": "3.0.4",
+    "@devexpress/dx-react-scheduler": "3.0.4",
+    "@devexpress/dx-react-scheduler-material-ui": "3.0.4",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
```

Comment:
This is the package.json for the backend... should be in the frontend. Just run the npm i commands you used in the ./frontend folder and it should fix it for you